packages/angular-workspace/example-client-app/src/app/customapp/customapp.component.ts

```diff
@@ -5,6 +5,9 @@ import { DrawerLocation, MenuItem, NimbleDialogDirective, NimbleDrawerDirective,
 import { NimbleTableDirective, TableRecordDelayedHierarchyState, TableRecord, TableRowExpansionToggleEventDetail, TableSetRecordHierarchyOptions } from '@ni/nimble-angular/table';
 import { NimbleRichTextEditorDirective } from '@ni/nimble-angular/rich-text/editor';
 import { BehaviorSubject, Observable } from 'rxjs';
+import type { MenuButtonColumnToggleEventDetail } from '@ni/nimble-angular/table-column/menu-button';
+
+type Color = 'Red' | 'Green' | 'Blue' | 'Black' | 'Yellow';
```

Comment:
nit: if you make `CustomAppComponent.possibleColor` a `static`, this could be: `type Color = typeof CustomAppComponent.possibleColor[number];`